<?xml version="1.0"?>
<launch>

    <node pkg="wamv_driver" type="status_light_interpretter.py" name="status_light_interpretter_node" >
      <remap from='yellow_light' to='/yellow_light_active' />
      <remap from='green_light' to='/green_light_active' />
      <remap from='blue_light' to='/blue_light_active' />
      <remap from='red_light' to='/red_light_active' />
      <!-- <remap from='system_mode' to='' /> -->
    </node>

    <node pkg="rosserial_python" type="serial_node.py" name="status_light_controller_node">
        <param name="port" value="/dev/ttyACM1" />
        <!-- <remap from='remote_control_active' to='' /> -->
        <!-- <remap from='autonomous_active' to='' /> -->
        <!-- <remap from='network_connection_active' to='' /> -->
        <!-- <remap from='heartbeat_string' to='' /> -->
    </node>

</launch>
